stable/nfs-server-provisioner/values.yaml

```diff
@@ -59,8 +65,8 @@ storageClass:
   parameters: {}
 
   mountOptions:
-    - vers=4.1
-    - noatime
+    - vers=3
+    - relatime
```

Comment:
```suggestion
```

Reply:
solved in https://github.com/helm/charts/pull/19260/commits/71de49c4ddab4058591049876d3f7a1819dbef4a